db/db_impl.cc

```diff
@@ -3828,6 +3828,8 @@ SuperVersion* DBImpl::InstallSuperVersionAndScheduleWork(
 
   // Whenever we install new SuperVersion, we might need to issue new flushes or
   // compactions.
+  cfd->current()->storage_info()->ComputeCompactionScore(*cfd->ioptions(),
+                                                         mutable_cf_options);
```

Comment:
Wow, this is tricky. Is there a way to create a new version and recompute the score, just to be safe?

We never change the score after the version is installed.